### ISSUE TITLE
'list' object has no attribute 'add'

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1650]
A got this error, with putting an underline character to a rst file:

```
#!python

# Sphinx version: 1.3b1
# Python version: 2.7.8
# Docutils version: 0.12 release
# Jinja2 version: 2.7.3
# Last messages:
#   done
#   building [mo]: targets for 0 po files that are out of date
#   building [html]: targets for 2 source files that are out of date
#   updating environment:
#   1 added, 2 changed, 0 removed
#   reading sources... [ 33&#37;] index
#   reading sources... [ 66&#37;] introduction
#   reading sources... [100&#37;] object_concept
#   
#   looking for now-outdated files...
# Loaded extensions:
#   sphinx.ext.coverage (1.3b1) from /usr/local/lib/python2.7/dist-packages/Sphinx-1.3b1-py2.7.egg/sphinx/ext/coverage.pyc
#   sphinx.ext.todo (1.3b1) from /usr/local/lib/python2.7/dist-packages/Sphinx-1.3b1-py2.7.egg/sphinx/ext/todo.pyc
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/Sphinx-1.3b1-py2.7.egg/sphinx/cmdline.py", line 246, in main
    app.build(opts.force_all, filenames)
  File "/usr/local/lib/python2.7/dist-packages/Sphinx-1.3b1-py2.7.egg/sphinx/application.py", line 254, in build
    self.builder.build_update()
  File "/usr/local/lib/python2.7/dist-packages/Sphinx-1.3b1-py2.7.egg/sphinx/builders/__init__.py", line 237, in build_update
    'out of date' &#37; len(to_build))
  File "/usr/local/lib/python2.7/dist-packages/Sphinx-1.3b1-py2.7.egg/sphinx/builders/__init__.py", line 260, in build
    updated_docnames.add(docname)
AttributeError: 'list' object has no attribute 'add'
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1650
- Originally reported by: İsmail Arılık
- Originally created at: 2014-12-14T09:36:32.198

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-12-14 08:50:39+00:00_

Thanks for reporting. 
Please try 1.3b2, I guess the issue will be solved.

**Comment by birkenfeld** [comment]
Should be fixed in 4d784bbb.
